presto-native-execution/presto_cpp/main/runtime-metrics/RuntimeMetricsReporter.cpp

```diff
@@ -0,0 +1,63 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "presto_cpp/main/runtime-metrics/RuntimeMetricsReporter.h"
+#include "presto_cpp/main/common/Configs.h"
+#include "presto_cpp/main/runtime-metrics/prometheus-reporter/PrometheusReporter.h"
```

Comment:
@karteekmurthys This line needs to be inside ifdef also
```
#ifdef PRESTO_ENABLE_PROMETHEUS_REPORTER
  if (auto reporter = std::dynamic_pointer_cast<prometheus::PrometheusReporter>(
          folly::Singleton<velox::BaseStatsReporter>::try_get())) {
    return reporter->getSerializedMetrics();
  }
#endif
```

Reply:
@karteekmurthys I think this is needed. Are you able to compile this PR with the Prometheus CMake flag disabled?
I think Amit suggested
```
#ifdef PRESTO_ENABLE_PROMETHEUS_REPORTER
#include "presto_cpp/main/runtime-metrics/prometheus-reporter/PrometheusReporter.h"
#endif
```